package.json

```diff
@@ -51,10 +51,11 @@
     "reset-css": "^2.2.0",
     "style-loader": "^0.18.2",
     "uglifyjs-webpack-plugin": "^2.1.3",
-    "webpack": "^4.29",
+    "webpack": "^4.29.0",
     "webpack-cli": "^3.3.5",
     "webpack-dev-server": "^3.7.2",
-    "ws": "^3.3.1"
+    "ws": "^3.3.1",
+    "yarn-deduplicate": "^1.1.1"
```

Comment:
Wow didn't know about this. What a tangle web we weave... with yarn.

Reply:
We use this in our spark repo's package.json and it's part of the pre-commit hook, which is nice!